reports-scheduler/build.gradle

```diff
@@ -255,8 +268,180 @@ testClusters.integTest {
         }
     }
     setting 'path.repo', repo.absolutePath
+
+//
+//    nodes.each { node ->
+//        def plugins = node.plugins
+//        def firstPlugin = plugins.get(0)
+//        plugins.remove(0)
+//        plugins.add(firstPlugin)
+//    }
+}
+
+// For job-scheduler and reports-scheduler, the latest opendistro releases appear to be 1.13.0.0.
+String bwcVersion = "1.13.0.0"
+String baseName = "reportsSchedulerBwcCluster"
+String bwcFilePath = "src/test/resources/bwc"
+
+2.times {i ->
+    testClusters {
+        "${baseName}$i" {
+            testDistribution = "ARCHIVE"
+            versions = ["7.10.2","1.3.0-SNAPSHOT"]
+            numberOfNodes = 3
+            plugin(provider(new Callable<RegularFile>(){
+                @Override
+                RegularFile call() throws Exception {
+                    return new RegularFile() {
+                        @Override
+                        File getAsFile() {
+                            return fileTree(bwcFilePath + "/job-scheduler/" + bwcVersion).getSingleFile()
+                        }
+                    }
+                }
+            }))
+            plugin(provider(new Callable<RegularFile>(){
+                @Override
+                RegularFile call() throws Exception {
+                    return new RegularFile() {
+                        @Override
+                        File getAsFile() {
+                            return fileTree(bwcFilePath + "/reports-scheduler/" + bwcVersion).getSingleFile()
+                        }
+                    }
+                }
+            }))
+            setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
+            setting 'http.content_type.required', 'true'
+        }
+    }
+}
+
+List<Provider<RegularFile>> plugins = [
+        provider(new Callable<RegularFile>(){
+            @Override
+            RegularFile call() throws Exception {
+                return new RegularFile() {
+                    @Override
+                    File getAsFile() {
+                        return fileTree(bwcFilePath + "/job-scheduler/" + project.version).getSingleFile()
+                    }
+                }
+            }
+        }),
+        provider(new Callable<RegularFile>(){
+            @Override
+            RegularFile call() throws Exception {
+                return new RegularFile() {
+                    @Override
+                    File getAsFile() {
+                        return fileTree(bwcFilePath + "/reports-scheduler/" + project.version).getSingleFile()
```

Comment:
there should be a way to not include the `1.3.0.0-SNAPSHOT` like other bwc PRs.

SQL is actually having the same issue, but SQL somehow doesn't have `bundlePlugin`. what alerting did might work here https://github.com/opensearch-project/alerting/blob/main/alerting/build.gradle#L204-L214

Reply:
I was following AD and they added that. But I see what you mean, maybe we can get rid of 1.3.0-snapshot for reports-scheduler, but we'll need the job-scheduler. I'll run some test

Reply:
changed to remove the reports-scheduler latest artifacts in `src/test/resources`